settings/dev.py

```diff
@@ -11,7 +11,7 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.postgresql_psycopg2",
+        "ENGINE": "settings.custom_databases.postgres_psycopg2",
```

Comment:
@burnerlee why do we need a custom DB wrapper?

Reply:
@Ram81 we need `django_prometheus.db.backends.postgresql_psycopg2` as our backend engine to support monitoring of database through prometheus. However django-prometheus library provides only `django_prometheus.db.backends.postgresql` as a built in engine in library. Thus to support our used engine in evalAI, i had to define a custom backend which has the properties of both `postgresql_psycopg2` and `prometheus`